README.md

```diff
@@ -44,6 +44,7 @@ path|status|notebooks|description
 [deploy-triton](tutorials/deploy-triton)|[![deploy-triton](https://github.com/Azure/azureml-examples/workflows/run-tutorial-dt/badge.svg)](https://github.com/Azure/azureml-examples/actions?query=workflow%3Arun-tutorial-dt)|[1.densenet-local.ipynb](tutorials/deploy-triton/1.densenet-local.ipynb)<br>[2.bidaf-aks-v100.ipynb](tutorials/deploy-triton/2.bidaf-aks-v100.ipynb)|learn how to efficiently deploy to GPUs using [triton inference server](https://github.com/triton-inference-server/server)
 [music-with-ml](tutorials/music-with-ml)|[![music-with-ml](https://github.com/Azure/azureml-examples/workflows/run-tutorial-mwm/badge.svg)](https://github.com/Azure/azureml-examples/actions?query=workflow%3Arun-tutorial-mwm)|[1.intro-to-magenta.ipynb](tutorials/music-with-ml/1.intro-to-magenta.ipynb)|learn how to create music with ML using [magenta](https://github.com/magenta/magenta)
 [using-dask](tutorials/using-dask)|[![using-dask](https://github.com/Azure/azureml-examples/workflows/run-tutorial-ud/badge.svg)](https://github.com/Azure/azureml-examples/actions?query=workflow%3Arun-tutorial-ud)|[1.intro-to-dask.ipynb](tutorials/using-dask/1.intro-to-dask.ipynb)<br>[2.eds-at-scale.ipynb](tutorials/using-dask/2.eds-at-scale.ipynb)|learn how to read from cloud data and scale PyData tools (numpy, pandas, scikit-learn, etc.) with [dask](https://github.com/dask/dask)
+[using-edge](tutorials/using-edge)|[![using-edge](https://github.com/Azure/azureml-examples/workflows/run-tutorial-ue/badge.svg)](https://github.com/Azure/azureml-examples/actions?query=workflow%3Arun-tutorial-ue)|[production-deploy-to-ase-gpu.ipynb](tutorials/using-edge/production-deploy-to-ase-gpu.ipynb)|learn how to use Edge device for deployment and inferencing
```

Comment:
description needs to be generated from `readme.py`

Reply:
That's probably outdated diff. That shouldn't be there, I renamed 'using-edge' and re-ran readme.py.